OTP-Server-P-Sample/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required (VERSION 3.1)
+project (P)
+set(projectName PingPong)
+find_package (Threads)
+include_directories(~/Research/P/Bld/Drops/Prt/include)
+add_executable(ping_pong
+	PingPong.c
+	PingPong.h
+	MainFunction.c
+	~/Research/P/Bld/Drops/Prt/include/ext_compat.h
+	~/Research/P/Bld/Drops/Prt/include/libhandler.h
+	~/Research/P/Bld/Drops/Prt/include/libhandler-internal.h
+	~/Research/P/Bld/Drops/Prt/include/Prt.h
+	~/Research/P/Bld/Drops/Prt/include/PrtConfig.h
```

Comment:
Rather than include each file, you should add directory. Please refer to the Cmake files in Drona and in P repo.